explorer-ui/src/components/ProfileView/Preferences/Preferences.jsx

```diff
@@ -145,27 +145,27 @@ export default function Preferences() {
     console.log("current location is: ", currLocation);
   }
 
-//   // Send the profile information to the database
-//   function handleSubmit(event) {
-//     console.log("submitted");
-//     event.preventDefault();
-//     const preferences = async () => {
-//       try {
-//         const res = await axios.post(
-//           `${config.API_BASE_URL}/profileInfo`,
-//           {
-//             preferenceInfo: preferenceInfo,
-//           },
-//           { maxContentLength: Infinity, maxBodyLength: Infinity }
-//         );
-//       } catch (err) {
-//         alert(err);
-//         console.log(err);
-//       }
-//     };
-//     preferences();
-//     console.log("preferences posted!");
-//   }
+// Send the profile information to the database
+  function handleSubmit(event) {
+    console.log("submitted");
+    event.preventDefault();
+    const preferences = async () => {
+      try {
+        const res = await axios.post(
+          `${config.API_BASE_URL}/profileInfo`,
+          {
+            preferenceInfo: preferenceInfo,
+          },
+          { maxContentLength: Infinity, maxBodyLength: Infinity }
+        );
+      } catch (err) {
+        alert(err);
+        console.log(err);
+      }
+    };
+    preferences();
+    console.log("preferences posted!");
```

Comment:
same as before ! outer fxn async to prevent having to call inner fxn after declaring